internal/storage/storageutil/custom_retry_test.go

```diff
@@ -0,0 +1,74 @@
+package storageutil
+
+import (
+	"io"
+	"net"
+	"net/url"
+	"testing"
+
+	. "github.com/jacobsa/ogletest"
+	"google.golang.org/api/googleapi"
+)
+
+func TestCustomRetry(t *testing.T) { RunTests(t) }
+
+type customRetryTest struct {
+}
+
+func init() { RegisterTestSuite(&customRetryTest{}) }
+
+func (t customRetryTest) TestGoogleAPIErrorRetry() {
```

Comment:
lets use consistent naming for the new tests we write in GCSFuse
say ShouldRetryWithGoogleApiErrorReturnsTrue
or
ShouldRetryReturnsTrueWithGoogleApiError